spec/ui/actions.rst

```diff
@@ -0,0 +1,259 @@
+#########
+Use Cases
+#########
+
+.. _ui_usecases_login:
+
+*****
+Login
+*****
+
+There is a number of scenarios related to login forms of ADCM.
+
+.. note::
+   Upgrade of ADCM and related events is out of scope for this use case.
+
+User/password login
+===================
+
+Just a simple use case of auth for user with username and password.
+
+**Goal**: successful login
+
+1. Go to ADCM
+2. Fill *Login* on :ref:`ui_elements_forms_login`
+3. Fill *Password* on :ref:`ui_elements_forms_login`
+4. Press button *Login* on :ref:`ui_elements_forms_login`
+
+Exceptions:
+
+2. User already logged to ADCM
```

Comment:
```suggestion
2. User has been already logged in to ADCM
```

Reply:
What about the case when the user has been already logged in but navigates to http://localhost:8004/login?
The case above is a bit weird since in this case for some reason user gets logged out without any warnings